docs/build.sbt

```diff
@@ -2,7 +2,8 @@ name := "anorm-docs"
 
 libraryDependencies ++= Seq(
   component("play-jdbc") % "test",
-  component("play-specs2") % "test"
+  component("play-specs2") % "test",
+  "com.h2database" % "h2" % "1.4.193"
```

Comment:
H2 used to be a dependency of Play and then it was added transitively here. But it was removed from Play and now should be added explicitly. Keep in mind this is the docs sub project.